### ISSUE TITLE
add scrollbar

### ISSUE THREAD

**Issue by wangmeng1991** [element-plus/element-plus#446]
```
Failed to compile.

./node_modules/element-plus/lib/index.esm.js
Module not found: Error: Can't resolve '@element-plus/scrollbar/src/index' in '/Users/lean/IdeaProjects1/vue3-demo/node_modules/element-plus/lib'

```

**Comment by JeremyWuuuuu** [comment]
We will export scroll bar to users in next alpha release.

**Comment by zouhangwithsweet** [comment]
> We will export scroll bar to users in next alpha release.

This error is found in `table`. There is a wrong import module

**Comment by zazzaz** [comment]
This should have fixed in `element-plus@1.0.1-alpha.9`

**Comment by wangmeng1991** [comment]
ok 👌
